src/Components/Nav/ProductDetail/ProductDetail.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import ContentLeft from "./ContentLeft/ContentLeft";
+import ContentRight from "./ContentRight/ContentRight";
+import "./ProductDetail.scss";
+
+class ProductDetail extends Component {
+  render() {
+    const { data } = this.props;
+    const leftData = {
+      imgUrls: data.imgUrls,
+      reviews: data.reviews,
+    };
+    const rightData = {
+      name: data.name,
+      options: data.options,
+      price: data.price,
+      type: data.type,
+    };
```

Comment:
```suggestion
const { imgUrls, reviews, name, options, price, type } = this.props.data;
const leftData = {
  imgUrls,
  reviews
}
```
와 같은 식으로 작성하실 수 있습니다 💪

Reply:
넵! 수정하겠습니다!